Documentation/CRDs/ceph-client-crd.md

```diff
@@ -41,3 +41,146 @@ spec:
 ### Prerequisites
 
 This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](../Getting-Started/quickstart.md).
+
+## Examples
```

Comment:
The header above "Use Cases" seems sufficient, so let's remove this header

Reply:
- [ ] remove "examples" header